src/modm/platform/dma/stm32/module.lb

```diff
@@ -186,6 +186,7 @@ def build(env):
     properties["dmaType"] = dma["type"]
     properties["dmaSignals"] = signal_names
     properties["dmaController"] = controller
+    properties["doubleBuffer"] = (dma["type"] in ["stm32-stream-channel"]) ## TODO: which other types support double buffer mode?
```

Comment:
I'll give it a try.

The DMA double buffer is currently not used anywhere with I2S, because that would make I2S more complicated or limit it to the devices with `stm32-stream-channel`/`stm32-mux-stream` DMA IP.
Works fine with the transfer complete interrupt, but maybe in the future someone can implement a mode with DMA double buffer and without interrupts.